src/main/java/com/greenfoxacademy/goddesstribesbackend/security/WebSecurityConfig.java

```diff
@@ -16,6 +17,15 @@
   @Autowired
   private RestAuthenticationEntryPoint restAuthenticationEntryPoint;
 
+  private static final String[] AUTH_WHITELIST = {
+
+      // -- swagger ui
+      "/swagger-resources/**",
+      "/swagger-ui.html",
+      "/v2/api-docs",
+      "/webjars/**"
+  };
+
```

Comment:
This is nice. 

We will talk about different environments later. (i.e. you do not want something like this out in production, ever.)